threadpool/core/src/main/java/cn/hippo4j/core/enable/BeforeCheckConfiguration.java

```diff
@@ -144,6 +144,33 @@ public BeforeCheckConfiguration.BeforeCheck dynamicThreadPoolBeforeCheckBean(@Au
                         }
                     }
 
+                    Map<String, String> polaris = properties.getPolaris();
+                    if (MapUtil.isNotEmpty(polaris)) {
+                        String namespace = polaris.get("namespace");
```

Comment:
Don't use magic variables

Reply:
Thank you very much! I am so sorry this is my first time to participate in open source projects, and I reresearched and modified the code after reading your comment.